packages/manager/modules/cloud-connect/src/details/service-keys/translations/Messages_fr_FR.json

```diff
@@ -3,7 +3,7 @@
   "cloud_connect_service_keys_status_active": "Actif",
   "cloud_connect_service_keys_status_cancelled": "Annulé",
   "cloud_connect_service_keys_status_terminated": "Terminé",
-  "cloud_connect_service_keys_status_toCheck": "Vérifier",
+  "cloud_connect_service_keys_status_toCheck": "Indisponible",
```

Comment:
Is it right ? I mean, "to check" means "Vérifier" in french.

Reply:
@JeremyDec 
The step `"ToCheck"` is a temporary procedure on backend performed from OVHcloud side by OVHcloud request. It is supposed to be `"Unavailable"` in english, so they have given equivalent word `"Indisponible"`